src/sentry/dynamic_sampling/feature_multiplexer.py

```diff
@@ -54,5 +58,12 @@ def get_user_biases(user_set_biases):
         return returned_biases
 
     @staticmethod
-    def get_supported_biases_ids():
+    def get_supported_biases_ids() -> Set[str]:
         return {bias["id"] for bias in DEFAULT_BIASES}
+
+    @classmethod
+    def get_user_bias(cls, bias_id: str, user_set_biases: Optional[List[Bias]]) -> Bias:
+        for bias in cls.get_user_biases(user_set_biases):
+            if bias["id"] == bias_id:
+                return bias
+        raise ValueError(f"{bias_id} is not in supported biases")
```

Comment:
This logic is incorrect because project_options may not store the entire options, it just stores the ones changed by a user so you need to factor the `get_supported_biases_ids` function somehow